draft-ietf-rtcweb-jsep.xml

```diff
@@ -3826,23 +3824,38 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>For each specified payload type that is also
                         supported by the local implementation, establish
-                        a mapping between the payload type ID and the
-                        actual media format. [TODO - Justin to add more
-                        to explain mapping.] If any indicated payload
-                        type is unknown, it MUST be ignored.  [TODO:
-                        should fail on answers]</t>
+                        a mapping between the specified payload type ID and the
+                        actual media format, as described in
+                        <xref target="RFC3264" />, Section 6.1.
+                        Specifically, this means that the
+                        implementation determines the payload type ID to
+                        be used in outgoing RTP packets when sending each
+                        specified payload type. If any indicated payload
+                        type is unknown, it MUST be ignored.</t>
 
                         <t>For each specified "rtx" media format,
                         establish a mapping between the RTX payload type
                         and its associated primary payload type, as
-                        described in <xref target="RFC4588" />. If any
-                        referenced primary payload types are not
+                        described in <xref target="RFC4588" />, Section 4.
+                        If any referenced primary payload types are not
                         present, this MUST result in an error.</t>
 
                         <t>For each specified fmtp parameter that is
                         supported by the local implementation, enable
                         them on the associated payload types.</t>
 
+                        <t>For each specified RTP header extension that is also
+                        supported by the local implementation, establish a
+                        mapping between the extension ID and
+                        URI, as described in <xref
+                        target="RFC5285" />, Section 5. Specifically, this means
+                        that the implementation determines the extension ID to
+                        be used in outgoing RTP packets when sending each
+                        specified header extension, and enables that extension
+                        for the outgoing streams associated with this m= section.
+                        If any indicated RTP header
```

Comment:
Note that I think the text around 'enabling' needs to move down to the 'handling an answer' section, since these things are only really enabled if the answer says so.